### PR TITLE
Migrate `personal_statement` column onto `application_choices`

### PR DESCRIPTION
**Pull request by inulty-dfe**
## Context

Add a database column of type `text` with the name `personal_statement` onto the `application_choices` table.

The column will later be filled with the value found in the associated `ApplicationForm#becoming_a_teacher` column. This also is a text field.

## Guidance to review

Nothing special here.

## Link to Trello card

https://trello.com/c/jaBPQVNF/1532-ca-add-a-personalstatement-attribute-to-applicationchoice

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Review by tomas-stefano — APPROVED** [review]
*Don't forget to talk with the Data insight teams about the field and check if should be on config/analytics or in the blocklist ;)

File: db/migrate/20230608145455_add_personal_statement_to_application_choice.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddPersonalStatementToApplicationChoice < ActiveRecord::Migration[7.0]
+  def change
+    add_column(:application_choices, :personal_statement, :text)
```
Comment (JR-G) [inline db/migrate/20230608145455_add_personal_statement_to_application_choice.rb:3]: apologies to nitpick 😬 but could we remove the brackets from this to remain consistent with other migrations 🙏

**Comment by inulty-dfe** [comment]
> *Don't forget to talk with the Data insight teams about the field and check if should be on config/analytics or in the blocklist ;)

James and I discussed this with the D&I team in mind and decided that its reasonable to go ahead and add it to the analytics list and when the data is being migrated later, the D&I team can then decide if they would like to block the column.

**Comment by inulty-dfe** [comment]
> *Don't forget to talk with the Data insight teams about the field and check if should be on config/analytics or in the blocklist ;)

James and I discussed this with the D&I team in mind and decided that its reasonable to go ahead and add it to the analytics list and when the data is being migrated later, the D&I team can then decide if they would like to block the column.

**Pull request merged**